### PR TITLE
style(*) code clean for m1

### PR DESCRIPTION
**Pull request by chronolaw**
Code clean for nginx style.

### CODE REVIEW

File: src/ngx_http_lua_kong_apple_m1.c [hunk 1]
```diff
@@ -20,11 +20,6 @@
 
 // macOS with M1 fixes, see: https://github.com/LuaJIT/LuaJIT/issues/205
```
Comment (dndx) [inline src/ngx_http_lua_kong_apple_m1.c:21]: Also should remove this or change the style to C89.

File: src/ngx_http_lua_kong_apple_m1.c [hunk 2]
```diff
@@ -39,21 +34,6 @@ typedef struct {
     char **errmsg;
 } ngx_shdict_get_t;
```
Comment (dndx) [inline src/ngx_http_lua_kong_apple_m1.c:35]: Let's move these into a separate header file.

**Review by dndx — COMMENTED** [review]
Basically LGTM, we should probably change the filename as well. Maybe `src/ngx_http_lua_kong_macos_shdict.c` is better.

File: src/ngx_http_lua_kong_apple_m1.c (hunk 1 above)
Comment (chronolaw) [reply src/ngx_http_lua_kong_apple_m1.c:21]: thanks

File: src/ngx_http_lua_kong_apple_m1.c (hunk 2 above)
Comment (dndx) [reply src/ngx_http_lua_kong_apple_m1.c:35]: Also, we should probably rename this struct to something like `ngx_http_lua_kong_shdict_get_t` or something similar. `ngx_shdict_get_t` seems to be a bit odd.

File: src/ngx_http_lua_kong_macos_arm64.h [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+#ifndef _NGX_HTTP_LUA_KONG_MACOS_ARM64_H_INCLUDED_
+#define _NGX_HTTP_LUA_KONG_MACOS_ARM64_H_INCLUDED_
+
+
+#include <ngx_config.h>
+#include <ngx_core.h>
+#include <ngx_http.h>
+
+
+/* macOS with Apple Silicon fixes, see: https://github.com/LuaJIT/LuaJIT/issues/205 */
```
Comment (dndx) [inline src/ngx_http_lua_kong_macos_arm64.h:27]:
```suggestion
/*
 * macOS with Apple Silicon fixes,
 * see: https://github.com/LuaJIT/LuaJIT/issues/205
 */
```

Just minor Nginx style correction.

**Review by bungle — APPROVED** [review]

**Pull request closed without merging**